client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -45,7 +45,6 @@ export default class ProductFormDetailsCard extends Component {
 
 		this.setName = this.setName.bind( this );
 		this.setDescription = this.setDescription.bind( this );
-		this.debouncedSetDescription = debounce( this.setDescription, 200 );
```

Comment:
When trying to track down this bug, this did jump out at me - what was the original reason for this logic?

Reply:
I'm not sure now, sorry.  It doesn't seem necessary from the testing I did though.

Reply:
Looks like it was added in here #14944 though i don't see any details around that specific bit. I think this might be some carry-over from patterns in the post editor maybe. But I agree if its not needed, it is great to remove it.